code/game/machinery/bank_machine.dm

```diff
@@ -33,6 +34,7 @@
 	radio.set_listening(FALSE)
 	radio.recalculateChannels()
 	synced_bank_account = SSeconomy.get_dep_account(account_department)
+	AddComponent(/datum/component/gps, "Forbidden Cash Signal")
```

Comment:
```suggestion
	if(!mapload)
		AddComponent(/datum/component/gps, "Forbidden Cash Signal")
```
I think it's better fit if the one mapped into the vault does not have the GPS signal, because why would it be "forbidden"

Reply:
To confuse people.